HW1/P4/P4_bfs.py

```diff
@@ -0,0 +1,17 @@
+# P4 bfs method
+
+def ssbfs(charname, charadjlist):
+    sc = charadjlist.context
+    result = sc.parallelize([(charname, 0)])  # the rdd that keep track with the visited notes
+    accum = sc.accumulator(0)  # seemingly playful
+    while True:
+        curvalue = accum.value  # the depth
+        waiting = result.filter(lambda kv: kv[1] == curvalue)  # the nodes of deepest level
```

Comment:
Since we had to use the accumulator, it could also be used here to get the size of the filtered set (same result though.)